src/index.ts

```diff
@@ -58,6 +90,11 @@ export type ImgixRect = {
     h: number;
 };
 
+// https://docs.imgix.com/apis/rendering/watermark/mark-align
+export type ImgixMarkAlignBase = 'top' | 'middle' | 'bottom' | 'left' | 'center' | 'right';
+// The api allows the user to combine the align values with a comma
+export type ImgixMarkAlign = `${ImgixMarkAlignBase},${ImgixMarkAlignBase}` | ImgixMarkAlignBase;
```

Comment:
The api allows the user to combine two values with comma like `top,middle`. One way of handling this is to make a Record<ImgixMarkAlignbase,boolean> or a tuple. But this is so much better UX for the end user imo. TS lsp will show a nice dropdown with fuzzy matching for all the possible values.

Reply:
I guess this notation would also allow `right right` or `middle,middle` which I think isn't valid values? since we're be targeting the same axis? One way to avoid that would just to be typing all possible values as part of the union? 

```ts
export type Markalign = "top-left" | "top-right" | ...
```

I think this way is a bit more type safe. WDYT?
Also it looks like `unset` is another possible value but perhaps we don't need it.

Reply:
right. This is the kind of thing where i'd spend hours trying to figure it out a generic way to remove duplicated with TS instead of manually writing it, only to end up realizing that manually typing would have been so much faster and easier

Reply:
lol it works with this https://github.com/unsplash/ts-imgix/pull/17/commits/5d090cd46a3a4caeede646aa8ea7494f183534d1